crates/types/src/config/worker.rs

```diff
@@ -567,3 +573,34 @@ impl SnapshotsOptions {
         super::data_dir("db-snapshots").join(partition_id.to_string())
     }
 }
+
+/// # Throttling options
```

Comment:
For eventual production use, we should better document the units that tokens represent.